### DISCUSSION TITLE
control_transfer returned error, index: 768, error: Resource temporarily unavailable, number: 11

### DISCUSSION THREAD

**Discussion by FarStryke21** [realsenseai/realsense-ros#2875] [Q&A]
control_transfer returned error, index: 768, error: Resource temporarily unavailable, number: 11

What is the warning indicating? Is there a way to mute it? 
(Using ROS1 wrapper for my L515. Got these warnings when running the launch script for the camera. The camera seems to be performing fine, but I keep getting this warning in the terminal)

**Comment by MartyG-RealSense** [comment]
Hi @FarStryke21  If this warning message is generating continuously in the ROS terminal then it indicates a communication problem between the camera and computer, such as a USB problem.  

You could mute the warnings by changing the **logging level**.

https://dev.intelrealsense.com/docs/troubleshooting#enable-librealsense-log

For example: 

**export LRS_LOG_LEVEL="INFO"**

Or you could send the warnings to a log file instead of the screen, as described at https://github.com/IntelRealSense/realsense-ros/issues/1185#issuecomment-621218705
